src/mpid/ch3/src/ch3u_port.c

```diff
@@ -1342,6 +1342,8 @@ static int SetupNewIntercomm( MPIR_Comm *comm_ptr, int remote_comm_size,
     intercomm->comm_kind    = MPIR_COMM_KIND__INTERCOMM;
     intercomm->local_comm   = NULL;
 
+    MPIR_Comm_set_session_ptr(intercomm, comm_ptr->session_ptr);
```

Comment:
ch3 does not support sessions. Do we need this commit ("ch3/port: add session ref counting to intercomm creation. ")?

Reply:
No, not really. Removed it.